lib/MethodFactory.js

```diff
@@ -107,11 +107,11 @@ module.exports = class MethodFactory {
       throw new TypeError('loglevelnext: Logger is undefined or invalid. Please specify a valid Logger instance.');
     }
 
-    for (const methodName of this.methods) {
+    this.methods.forEach((methodName) => {
```

Comment:
The polyfill didn't place nicely with the Babel transformation code that results from the `for..of` so I changed this to a `forEach`. IE 11 was thowing an `Object expected` error.